target-data/get_target_data_hubverse.R

```diff
@@ -162,12 +163,16 @@ create_oracle_output_wk_inc <- function(time_series_target) {
     data.frame(target = "wk inc flu hosp"),
     time_series_target[c("target_end_date", "location", "observation", "as_of")]
   )
-  colnames(oracle_output_wk_inc) <- c("target", "target_end_date", "location", "oracle_value", "as_of")
+  colnames(oracle_output_wk_inc) <- c(
+    "target", "target_end_date", "location", "oracle_value", "as_of")
   oracle_output_wk_inc <- oracle_output_wk_inc |>
     dplyr::cross_join(
       # add a row for each horizon defined in the modeling task
       # (except horizon -1, which is not used for scoring/viz)
       data.frame(horizon = 0:3)
+    ) |>
+    dplyr::mutate(
+      output_type = "quantile",
     )
```

Comment:
The flusight hub has an optional "sample" output type that needs to be included.

I'm presenting two options in subsequent comments depending on what you want to do. Choose one.

Reply:
Option 1: deal with the horizon later

```suggestion
      expand.grid(horizon = 0:3, output_type = c("quantile", "sample"))
    )
```

Reply:
Option 2: yeet the horizon:


```suggestion
      data.frame(output_type = c("quantile", "sample"))
    )
```